packages/wrangler/src/__tests__/config/configuration.test.ts

```diff
@@ -6645,6 +6645,81 @@ describe("experimental_readRawConfig()", () => {
 	);
 });
 
+describe("BOM (Byte Order Marker) handling", () => {
+	runInTempDir();
+
+	it("should remove UTF-8 BOM from TOML config files", () => {
+		const configContent = `name = "test-worker"
+compatibility_date = "2022-01-12"`;
+
+		fs.writeFileSync("wrangler.toml", "\uFEFF" + configContent, "utf-8");
```

Comment:
I think we should create these files directly from Buffers to ensure we have exactly the right bytes.